pkg/kubelet/cm/cpumanager/policy_static_test.go

```diff
@@ -145,13 +149,14 @@ func TestStaticPolicyStart(t *testing.T) {
 	}
 	for _, testCase := range testCases {
 		t.Run(testCase.description, func(t *testing.T) {
-			p, _ := NewStaticPolicy(testCase.topo, testCase.numReservedCPUs, cpuset.New(), topologymanager.NewFakeManager(), nil)
+			logger, _ := ktesting.NewTestContext(t)
+			p, _ := NewStaticPolicy(logger, testCase.topo, testCase.numReservedCPUs, cpuset.New(), topologymanager.NewFakeManager(), nil)
 			policy := p.(*staticPolicy)
 			st := &mockState{
 				assignments:   testCase.stAssignments,
 				defaultCPUSet: testCase.stDefaultCPUSet,
 			}
-			err := policy.Start(st)
+			err := policy.Start(context.Background(), st)
```

Comment:
Instead of creating new context with `context.Background()` you can use test context returned by the `ktesting.NewTestContext` above.

Reply:
oops, sorry. I've already suggested that before.